recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.5" %}
+{% set version = "2.6" %}
 
 package:
   name: gsl
   version: {{ version }}
 
 source:
   url: http://ftp.gnu.org/gnu/gsl/gsl-{{ version }}.tar.gz
-  sha256: 0460ad7c2542caaddc6729762952d345374784100223995eb14d614861f2258d
+  sha256: b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8
   patches:
     # diff from source distribution to https://github.com/ampl/gsl/tree/v2.4.0
     - patch_cmake_for_windows.diff  # [win]
```

Comment:
The patch no longer applies but generating it again with a diff to 2.4 seems odd.
@isuruf am I missing something obvious here? Can we build `gsl 2.6` without the need for that patch?

Reply:
No, you have to wait for the gsl cmake fork to update it for you.
If you feel adventurous, you can use only this small part, https://github.com/conda-forge/gsl-feedstock/blob/master/recipe/patch_cmake_for_windows.diff#L1235-L1368 and use `cccl` to make `cl.exe` work with autotools.

Reply:
> If you feel adventurous, you can use only this small part, 

No thank you :smile: 

I'm "away" for 2 weeks. Maybe once I get back  I can play with this.